src/components/TextField/TextField.scss

```
@@ -310,3 +310,8 @@ $stacking-order: (
     margin-top: 0;
   }
 }
+
+.Monospaced {
```

Bit of a nitpick but our variations are usually lowerCamelCase and I'm not sure if we have conventions but I'd prefer if we matched the property name here as well

```suggestion
.monospace {
```